include/RAJA/policy/cuda/policy.hpp

```diff
@@ -122,6 +123,19 @@ struct cuda_reduce_base
                                                 RAJA::Platform::cuda> {
 };
 
+//
+// Cuda atomic policy for using cuda atomics on the device and
+// the provided Policy on the host
+//
+template<typename host_policy>
+struct cuda_atomic_explicit{};
```

Comment:
Just curious, why have we moved this here? Was there a reason it couldn't be kept in policy/cuda/atomic.hpp?

Reply:
If I'm remembering correctly, @davidbeckingsale and I had to move omp_atomic or some other atomic policy out of its atomic.hpp file and into its policy.hpp file otherwise we would need more macro guards due to including duplicate function definitions. Once we did it with one, it made sense to do it with the others. I know for some dependencies it sort of "cascaded" once we moved one or two as well.